CMakeLists.txt

```diff
@@ -511,7 +516,7 @@ include_directories(SYSTEM velox/external/duckdb/tpch/dbgen/include)
 
 # these were previously vendored in third-party/
 if(NOT VELOX_DISABLE_GOOGLETEST)
-  set(gtest_SOURCE BUNDLED)
+  set(gtest_SOURCE AUTO)
```

Comment:
nit: curious if this is related to conda.

Reply:
Yes to enable use of gtest from conda vs always building it from source. Why only set BUNDLED because it was previously vendored if it is necessary that we use that specific version for some reason we can revert this change of course.